sfcasts/field-rendering.md

```diff
@@ -1,51 +1,19 @@
-So you can see there's lots of different ways to play with these fields to get 'em in
-just the right spot. All right. So now that full name's gone from the form. We need
-to put first name and last name back. So let's say yield and I'll just use the lazy
-field call, call new way.
-
-First name, last name that we'll get it to show up here perfectly then back on the
-list page, we don't want those. You guys know what to do. So there's a nice little
-method for this called only on forms. So I'll copy that and repeat that for last name
-and now perfect. Then finally, I'm gonna do the same thing for created at I like
-having created at, on the list. I do not like having created at inside of the form
-because that's really supposed to be set automatically. Sit down here. We will say
-hide on form, be beautiful. So that was fairly easy and fairly simple. Next, I wanna
-dive a little bit further into the fields. We're gonna take one of these fields and
-configure its form type in a different way. Now, as we do, we're gonna learn about a
-concept called field configurator.
+As we discussed earlier, `configureFields()` controls how the fields are rendered on both the list page *and* the form pages. That leaves us with a situation that isn't exactly ideal. For example, we don't want an ID field on our form. We only want to render it on the index page.
 
+To fix that, there are a bunch of really useful methods on these field classes we can utilize. For instance, we can call `->onlyOnIndex()`, and just like that, it's gone from the form page, but we still have it on the index page. As you're playing with these methods, I want to invite you to dive in and check out the code behind the scenes, because it will help you understand EasyAdmin better.
+
+Methods like `->onlyOnIndex()` will give us tons of control, but also notice that `configureFields()` is passed the `$pageName`, which are things like "index", "detail", or "edit". So in the end, you can always just put `if` statements inside of here and dynamically return or *don't* return different things from this method. 
+
+The other problem on our form is that we have this `fullName()` method. In the database, we have a "First Name" field and a "Last Name" field. It's kind of nice to be able to render them as a "Full Name" on the index page, but ultimately, when we go to the form, we'll probably want to control their first name and last name separately.
+
+At the moment, this doesn't work. If I change something and then submit it, I get an error. It says "Could not determine access type for property "fullName"...". This is because, inside of our `User.php` class, I have `getFullName()`, but I don't have a `setFullName()` (and I don't really want one). The *point* is that over inside `configureFields()`, we can take that configure `fullName` and say `->onlyOnIndex();`. Now we'll have "Full Name" on our index, but we won't have "Full Name" on our form.
```

Comment:
When I read verbatim messages from the screen, I like to put them in a block quote. So something like:

```raw
It says:

> Could not determine...

This is because...
```